packages/read-model/tests/Unit/Product/Action/ProductActionViewRepositoryTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ReadModelBundle\Unit\Product\Action;
+
+use PHPUnit\Framework\TestCase;
+use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Product\Collection\ProductCollectionFacade;
+use Shopsys\FrameworkBundle\Model\Product\Product;
+use Shopsys\ReadModelBundle\Product\Action\ProductActionView;
+use Shopsys\ReadModelBundle\Product\Action\ProductActionViewFactory;
+use Shopsys\ReadModelBundle\Product\Action\ProductActionViewRepository;
+
+class ProductActionViewRepositoryTest extends TestCase
+{
+    public function testGetForProducts(): void
+    {
+        $productActionViewFactory = new ProductActionViewFactory();
+
+        $domainConfig = new DomainConfig(1, 'http://webserver:8080/', 'shopsys', 'en');
+
+        $domain = $this->createMock(Domain::class);
```

Comment:
I suggest to change the `$domain` to class property and setup it in `setup`

Reply:
I extracted the domain mock creation into the separate method as it is not needed to be class property.